python.sh

```diff
@@ -115,5 +115,10 @@ alibuild-generate-module --bin --lib > "$MODULEFILE"
 cat >> "$MODULEFILE" <<EoF
 setenv PYTHONHOME \$PKG_ROOT
 prepend-path PYTHONPATH \$PKG_ROOT/lib/python/site-packages
-setenv SSL_CERT_FILE  [exec \$PKG_ROOT/bin/python3 -c "import certifi; print(certifi.where())"] 
+if { [module-info mode load] } {
```

Comment:
Where does "module-info" come from? It does not seem to exists on lxplus.

Reply:
https://modules.readthedocs.io/en/latest/modulefile.html#mfcmd-module-info

I tested it on CC7 with modules `3.2.10` and `4.6.1`...